packages/neuron-wallet/src/models/asset-account-info.ts

```diff
@@ -263,6 +272,12 @@ export default class AssetAccountInfo {
     return !!exist
   }
 
+  public isDefaultLockScript(script: Script): boolean {
```

Comment:
https://github.com/nervosnetwork/neuron/blob/1e38bf860b46e5358081aefb2b816ceb52d32cb9/packages/neuron-wallet/src/models/system-script-info.ts#L77

Reply:
Ok, I will replace it.